source/getting-started/use_cases/file_integrity.rst

```diff
@@ -98,9 +98,29 @@ Expand the output to see an example response of this use case.
 
 In the *Integrity monitoring* module of the Wazuh dashboard, users can view all of the details of the alerts triggered and find a comprehensive summary of changes in files and registry keys. 
 
+.. raw:: html
```

Comment:
These images need to be edited. I can see the black line from the header you removed.

Also, I can tell images in this class have different sizes. Please, ask the design team to adjust them properly. If this is too complicated for them to change, we can discuss it we should leave them as it is. 

![image](https://user-images.githubusercontent.com/79285586/141321597-52d8c798-1d9b-4ac8-97b3-670f78b13ce3.png)